### ISSUE TITLE
[BITV]: aria-label attribute cannot be used on a div with no valid role attribute.

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#42626]
![Screenshot from 2024-01-08 14-15-14](https://github.com/nextcloud/server/assets/6078378/7835b678-cfda-4954-81bc-08e6c180a2e1)

Implementation 

```<div data-v-ee6f34b5="" aria-label="MIME type image/png" class="_file-picker__file-icon_1vgv4_5" style="background-image: url(...);"></div>```

```<div data-v-ee6f34b5="" aria-label="MIME type text/markdown" class="_file-picker__file-icon_1vgv4_5" style="background-image: url(...);"><!----></div>```

> Note: aria-label is intended for use on interactive elements, or elements made to be interactive via other ARIA declarations, when there is no appropriate text visible in the DOM that could be referenced as a label
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label

Question: do we need this `aria-label` at all? @artonge @ShGKme ?

**Comment by ShGKme** [comment]
> Question: do we need this `aria-label` at all? @artonge @ShGKme ?

I think, this would be good to have an image with a description here. This image helps to understand what type of file is here.

But I don't think, this is required (file name is still available). And that would not be easy to implement as any app can register new icon image for a new file type.

So I'd just remove it.

**Comment by JuliaKirschenheuter** [comment]
closing as done
